### DISCUSSION TITLE
Is it possible specify GPU?

### DISCUSSION THREAD

**Discussion by FFAMax** [tinygrad/tinygrad#7379] [Q&A] [answered]
Hello, Team.
Could you please advice, how can utilize all GPU in the system or run multiple instances with specifying exact GPU so each instance will have dedicated? Thanks

**Comment by nimlgen** [comment, marked as answer]
For cuda use `CUDA_VISIBLE_DEVICES` (I see you mentioned `nvidia-smi` in the original issue)

**Comment by FFAMax** [comment]
So, just verified, to use both GPU need run something like 
CUDA_VISIBLE_DEVICES=0 exo  --node-id=1 --node-port=8001 --listen-port=65001
CUDA_VISIBLE_DEVICES=1 exo  --node-id=2 --node-port=8002 --listen-port=65002
